fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -1928,6 +1928,7 @@ public long loadResources(DataInputStream in, long checksum) throws IOException
 
     public long loadResourceGroups(DataInputStream in, long checksum) throws IOException {
         resourceGroupMgr = ResourceGroupMgr.read(in);
+        resourceGroupMgr.initResourceGroup();
```

Comment:
Is this line belong to this PR?

Reply:
Yes, after queue property is loaded from image, we need to create QueryQueue for group.